iOSApp/Screens/Main/HorizontalRailsTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  HorizontalRailsTableViewCell.swift
+//  iOSApp
+//
+//  Created by Inas Ahmic on 31. 3. 2021..
+//
+
+import UIKit
+
+class HorizontalRailsTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(UINib(nibName: "HorizontalRailsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "HorizontalRailsCollectionViewCell")
```

Comment:
move this into separate function like `setupCollectionView()` and call it from didSet from collectionView.

`
@IBOutlet weak var collectionView: UICollectionView! {
didSet { self.setupCollectionView() }
}
`